packages/core/src/utils/dynamicContent.ts

```diff
@@ -0,0 +1,22 @@
+import dynamicContent from 'src/customizations/src/dynamicContent'
+
+export async function getDynamicContent({ pageType }: { pageType: string }) {
+  // Checking if the fetch function corresponding to the type of page exists (home or LP slug)
+  const fetchFunction = dynamicContent[pageType]
+  let serverData = null
```

Comment:
Nice! I added some changes [here](https://github.com/vtex/faststore/pull/2242/commits/4f85b73d4edc0e128df768a10090f395ba73ae84). 
The `console.warn` is not necessarily an error in this case because it could be ok if the fetch function doesn't exist when dynamic content is not implemented (if the page don't need additional dynamic content). But we improved the messages in the catch block as well. Thanks!